xgi/drawing/draw.py

```diff
@@ -1664,160 +1708,465 @@ def draw_dihypergraph(
     See Also
     --------
     draw
-    draw_nodes
-    draw_node_labels
+    draw_multilayer
 
     """
-    if not isinstance(DH, DiHypergraph):
-        raise XGIError("The input must be a DiHypergraph")
+
+    is_directed = False
+    if isinstance(H, DiHypergraph):
+        from ..convert import to_hypergraph
+
+        DH = H.copy()
+        H = to_hypergraph(DH)
+        is_directed = True
+
+    if not isinstance(H, Hypergraph):
+        raise XGIError("The input must be a Hypergraph")
 
     settings = {
-        "min_node_size": 5,
-        "max_node_size": 30,
         "min_node_lw": 0,
         "max_node_lw": 5,
-        "min_lines_lw": 0,
-        "max_lines_lw": 50,
+        "min_node_size": 5,
+        "max_node_size": 30,
+        "min_edge_marker_lw": 0,
+        "max_edge_marker_lw": 5,
+        "min_edge_marker_size": 0,
+        "max_edge_marker_size": 5,
+        "min_dyad_lw": 1,
+        "max_dyad_lw": 10,
         "min_source_margin": 0,
         "min_target_margin": 0,
     }
 
     settings.update(kwargs)
 
-    # convert to hypergraph in order to use the augmented projection function
-    H_conv = convert.to_hypergraph(DH)
+    node_settings = {
+        "min_node_lw": settings["min_node_lw"],
+        "max_node_lw": settings["max_node_lw"],
+        "min_node_size": settings["min_node_size"],
+        "max_node_size": settings["max_node_size"],
+    }
 
-    ax, _ = _draw_init(H_conv, ax, True)
+    edge_marker_settings = {
+        "min_node_lw": settings["min_edge_marker_lw"],
+        "max_node_lw": settings["max_edge_marker_lw"],
+        "min_node_size": settings["min_edge_marker_size"],
+        "max_node_size": settings["max_edge_marker_size"],
+    }
 
-    if not max_order:
-        max_order = max_edge_order(H_conv)
+    if not pos:
+        pos = bipartite_spring_layout(H)
 
-    # set default colors
-    if lines_fc is None:
-        lines_fc = H_conv.edges.size
+    node_pos, edge_pos = pos
```

Comment:
Since the `pos` format is different from other functions, should we raise an explicit error after checking if it's a two-dict tuple? Right now a single dict would give a generic error like "too much to unpack"

Reply:
I added a case checking this.